core/templates/pages/exploration-player-page/layout-directives/progress-nav.directive.html

```diff
@@ -34,12 +34,12 @@
                  translate="I18N_INTERACTIONS_SUBMIT">
       </md-button>
     </div>
-    <continue-button ng-if="shouldContinueButtonBeShown()"
-                     focus-on="<[::CONTINUE_BUTTON_FOCUS_LABEL]>"
-                     is-learn-again-button="isLearnAgainButton()"
-                     ng-click="onClickContinueButton()"
-                     class="d-block oppia-autofocus">
-    </continue-button>
+    <oppia-continue-button ng-if="shouldContinueButtonBeShown()"
+                           [focusLabel]="CONTINUE_BUTTON_FOCUS_LABEL"
+                           [is-learn-again-button]="isLearnAgainButton()"
```

Comment:
Just wanted to check, have you tested that the "learn again" button shows correctly?

Reply:
I have updated the "Proof that changes are correct" showing `learn again` button working correctly. Please take a look.